### ISSUE TITLE
sw_thumbnails should support empty alt attributes for a11y

### ISSUE THREAD

**Issue by tobiasberge** [shopware/shopware#8228]
### Technical TODO

Currently, the `attributes` config of `sw_thumbnails` does not support empty attributes. Whenever a given attribute has an empty value, it will be completely skipped.

This is fine in most cases, but for accessibility it is common to set an empty `alt=""` attribute to declare an image as decorative. A missing alt attribute is not recommended. 

We should check if we can implement an edge-case for alt attributes so they can be empty. In the meantime `role: presentation` can be used as alternative.

```twig
{% sw_thumbnails 'my-decorative-image' with {
    media: lineItem.cover,
    sizes: {
        default: '100px',
    },
    attributes: {
        alt: ''
    }
} %}

<!-- This should create an empty alt attribute -->
<img srcset="" alt="">

<!-- But the attribute is omitted currently -->
<img srcset="">
```

**Comment by Be-njamin** [comment]
The new flag: `isDecorative` that can be set on images in the cms, will result in: `alt=" "` which does not result in a valid attribute value for decorative images.

![Image](https://github.com/user-attachments/assets/9f0b7082-fb53-42f1-acef-24ba664d5867)
![Image](https://github.com/user-attachments/assets/d763934c-61f6-4c70-ad6a-3f4f0f4f7fa8)

**Comment by tobiasberge** [comment]
Hi @Be-njamin,

in which version did you see this behaviour? In the current trunk the alt attribute is passed as `alt: ''` (without the space). But this removes the alt attribute altogether because all empty attributes are omitted by `sw_thumbnail`.

I created a [PR](https://github.com/shopware/shopware/pull/10557) what allows empty `alt=""` attributes so the logic from `cms-element-image.html.twig` and `isDecorative` takes affect.
